conf/evolutions/bakery_db/1.sql

```diff
@@ -13,6 +13,36 @@ CREATE TABLE IF NOT EXISTS Product
     CONSTRAINT "bakery-inventory_pkey" PRIMARY KEY (id)
 );
 
+CREATE OR REPLACE FUNCTION update_modified_column()
+RETURNS TRIGGER AS $$
+BEGIN
+    NEW.updated_at = now();;
+    RETURN NEW;;
+END;;
+$$ language 'plpgsql';
+
+CREATE TRIGGER update_product_modtime
+    BEFORE UPDATE ON Product
+    FOR EACH ROW
```

Comment:
so for each an every single row, you will trigger an update in the `updated_at` column? doesn't sound right, we only want updates in the affected row

Reply:
This actually only affects the rows getting updated only, it is future proof if there will ever be a new endpoint that updates multiple products. But since we only update one row, this works fine as is, and I did check the database to make sure, it works correctly.

Reply:
Can you add a test for this to ensure the correct usage? Saying "I tested this so I know" is not acceptable as a general development practice. You need to add a unit test, integration test or some other evidence (even a sequence of screen shots or a video/gif!)

Reply:
You are right, here.
This is using postman, I made a POST and gave it a JSON request to update specifically Sprinkled Donuts
<img width="846" alt="Screen Shot 2022-11-23 at 11 21 48 AM" src="https://user-images.githubusercontent.com/60335436/203610030-2dde0602-f249-4bb7-bc02-b5b2fdc12134.png">
And if we look at the database now
<img width="996" alt="Screen Shot 2022-11-23 at 11 23 04 AM" src="https://user-images.githubusercontent.com/60335436/203610409-1992eaad-1390-456e-8fe0-f2cfa73089d0.png">
You can see that only Sprinkled Donuts has updated at field set to the time I actually updated that row.